### PR TITLE
Drop legacy 2020 dated tags, update 2020 to 2021

### PR DESCRIPTION
**Pull request by merlinorg**

### CODE REVIEW

File: leaderboards/opmdays.yml [hunk 1]
```diff
@@ -21,6 +21,6 @@ query: |
     sum(R.distance) as distance
   from lbd_athletes A
     join rides R on R.athlete_id=A.id
-  where date(R.start_date) in ('2020-01-26')
+  where date(R.start_date) in ('2021-01-18')
```
Comment (hozn) [inline leaderboards/opmdays.yml:24]: I suppose this date really can't be known until later in the competition (not that this really matters here anyway; removing any 2020 references seems appropriate).

**Review by hozn — APPROVED** [review]
While I can't comment on how exhaustive this is, these changes look good to me.

**Comment by hozn** [comment]
Looks like the CI pipeline is broken, though.  Perhaps related to recent Docker Hub changes?

**Review by obscurerichard — APPROVED** [review]

**Comment by obscurerichard** [comment]
> 
> 
> Looks like the CI pipeline is broken, though. Perhaps related to recent Docker Hub changes?

I fixed the CI pipeline for Docker Hub builds, something got pickier for sure, but I think it may have been a [pip update where the behavior dealing with incompatible versions changed](https://stackoverflow.com/questions/63277123/what-is-use-feature-2020-resolver-error-message-with-jupyter-installation-on).

**Pull request merged**